src/compas_timber/ghpython/components/CT_Joint_Options_LButt/code.py

```diff
@@ -15,6 +15,6 @@ def RunScript(self, small_beam_butts, modify_cross, reject_i):
         if reject_i is not None:
             args["reject_i"] = reject_i
 
-        options = JointOptions(LButtJoint, **args)
+        options = JointOptions(LButtJoint, ["main_beam", "cross_beam"], **args)
```

Comment:
don't we need to add an argument to `JointOptions.__init__()` so that it accepts now the least of beam names?

Reply:
somehow that got deleted in a merge. Now it's back!